docs/installation-and-setup.md

```diff
@@ -58,12 +58,17 @@ return [
                  */
                 'follow_links' => false,
 
+                /*
+                 * Determines if it should avoid unreadable folders.
```

Comment:
It might be good the explain what would happen if this is set to `true` and an unreadable directory is processed. Will an exception be thrown?

Reply:
I've just copied the real config to here as it was out of sync. 😅